backend/api/Database/Models/IsarStep.cs

```diff
@@ -9,15 +9,18 @@ namespace Api.Database.Models
     public class IsarStep
     {
         [Key]
+        [Required]
```

Comment:
Keys are always required in SQL, but adding this tag will make it impossible to use SQLite because it will require POSTing of new entities to be filled in if i remember correctly.